parmed/amber/netcdffiles.py

```diff
@@ -291,13 +295,17 @@ def cell_angles(self, stuff):
 
     @property
     def box(self):
-        leng, ang = self.cell_lengths, self.cell_angles
-        return np.concatenate((leng, ang))
+        if self.hasbox:
```

Comment:
`box` here is just a composite value.  Instead, put the protection I described for velocities in for `cell_lengths` and `cell_angles`.  That way this fix gets applied to those descriptors as well.

Reply:
I think we still need to make sure `self.cell_lengths` and `self.cell_angles` are not `None` here.

Reply:
agreed